core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.html

```diff
@@ -25,16 +25,25 @@
         <ng-template pTemplate="caption">
             <div class="flex justify-content-between dot-pages-listing-header">
                 <div class="dot-pages-listing-header__inputs">
-                    <input
-                        class="dot-pages-listing-header__filter-input"
-                        [placeholder]="'Type-To-Search' | dm"
-                        [value]="vm.keywordValue"
-                        (input)="filterData($event.target.value)"
-                        data-testid="dot-pages-listing-header__keyword-input"
-                        type="text"
-                        pInputText
-                        dotAutofocus
-                    />
+                    <div class="p-input-icon-right">
+                        <input
+                            class="dot-pages-listing-header__filter-input"
+                            #input
+                            [placeholder]="'Type-To-Search' | dm"
+                            [value]="vm.keywordValue"
+                            (input)="filterData($event.target.value)"
+                            data-testid="dot-pages-listing-header__keyword-input"
+                            type="text"
+                            pInputText
+                            dotAutofocus
+                        />
+                        <i
+                            class="pi pi-times"
+                            *ngIf="input.value.length"
+                            (click)="filterData('')"
+                            data-testid="dot-pages-listing-header__keyword-input-clear"
+                        ></i>
```

Comment:
Why this is not a button?

Reply:
This is how PrimeNG adds icons to the inputs, I added the click event to clear the `filterData`. It should be a button for sure. But I'm not sure if that approach is going to break the styles that PrimeNG adds to position the icon correctly

Reply:
Ok add `role="button"` for a11y puerposes.